images/win/scripts/Installers/Install-Selenium.ps1

```diff
@@ -9,11 +9,12 @@ $seleniumFileName = "selenium-server-standalone.jar"
 
 New-Item -ItemType directory -Path $seleniumDirectory
 
-# Download Selenium
-$url = "https://api.github.com/repos/SeleniumHQ/selenium/releases/latest"
-[System.String] $seleniumReleaseUrl = (Invoke-RestMethod -Uri $url).assets.browser_download_url -match "selenium-server-standalone-.+.jar"
+# Download Selenium 3.141.59, since 4.* contains some breaking changes
+#$url = "https://api.github.com/repos/SeleniumHQ/selenium/releases/latest"
+#[System.String] $seleniumReleaseUrl = (Invoke-RestMethod -Uri $url).assets.browser_download_url -match "selenium-server-standalone-.+.jar"
+$seleniumDownloadUrl = "https://selenium-release.storage.googleapis.com/3.141/selenium-server-standalone-3.141.59.jar"
 
```

Comment:
I believe we should use https://api.github.com/repos/SeleniumHQ/selenium/releases and retrieve the latest 3.* version because version 3 will also receive some updates judging by 3.150 pre-release.